src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/LeggTilEllerFjernBrevmottakere/LeggTilBrevmottakerModal.tsx

```diff
@@ -50,10 +54,33 @@ export const LeggTilBrevmottakerModal: React.FC<Props> = ({
     åpenBehandling,
 }: Props) => {
     const { vurderErLesevisning } = useBehandling();
+    const { bruker, skjemaHarValgtFortroligBarn } = useFagsakContext();
     const erLesevisning = vurderErLesevisning();
 
     const heading = utledHeading(åpenBehandling.brevmottakere.length, erLesevisning);
 
+    const brukerData = hentDataFraRessurs(bruker);
```

Comment:
Ettersom denne ikke inneholder en manuellt lagt til person, burde den også hente den informasjon? Eller burde kallet som henter RestUtvidetBehandling returnere adressebeskyttelseGradering noe sted?

Reply:
Har vært luftet tanker rundt potensielle endringer i backend for å få tilstrekkelig data - men etter drøfting med Tor og Daphne ble det besluttet og benytte de data som foreligger og validere basert på de - og opprette et eget feil/mangler kort for prioritering av videre rettelser - som nødvendigvis trenger mer data på et senere tidspunkt..
Ref: https://favro.com/organization/98c34fb974ce445eac854de0/1844bbac3b6605eacc8f5543?card=NAV-13331